### DISCUSSION TITLE
Update language list

### DISCUSSION THREAD

**Discussion by Prasanta-Hembram** [immersive-translate/immersive-translate#2245] [Ideas]
### 新功能 | New Feature

1. Add new languages which are not updated in this extension (Google Translate).
2.
3.

### 为什么您想要添加这些功能 | Why you want to add these features

1. Allow users to take advantage in their mother language.
2. Google recently added many new languages which are still not updated in this extension.
3.

**Comment by theowenyoung** [comment]
Hello, what language is it in and can you give us an example? Our extension uses a list from Google Translate for Google Chrome, you can try to use Google Chrome's own translation service in Google Chrome to try to see if the language is supported. If it is supported then we can add it for you.

**Reply by Prasanta-Hembram** [reply]
I think they haven't yet updated in Google Chrome's language list. Now Google translate supports 249 languages (source: https://en.m.wikipedia.org/wiki/Google_Translate#Supported_languages). Can be tried and tested by opening translate.google.com website and using Google translate android application.
